app/src/main/java/ayds/spotisong/aurora/otherdetails/model/repository/OtherInfoRepositoryImp.kt

```diff
@@ -0,0 +1,30 @@
+package ayds.spotisong.aurora.otherdetails.model.repository
+
+import ayds.spotisong.aurora.otherdetails.model.repository.external.OtherInfoExternalService
+import ayds.spotisong.aurora.otherdetails.model.ArtistWrapper
+import ayds.spotisong.aurora.otherdetails.model.repository.local.DataBase
+import ayds.spotisong.aurora.song.model.Song
+private const val EXISTE_EN_DB_TAG = "[*]"
+internal class OtherInfoRepositoryImp(service : OtherInfoExternalService) : OtherInfoRepository{
+    private var service = service
```

Comment:
No hace falta que sea var. Se puede resolver en el constructor
```suggestion
   internal class OtherInfoRepositoryImp(val service : OtherInfoExternalService)
```